scaleway-core/scaleway_core/api.py

```diff
@@ -131,13 +144,7 @@ def _request(
 
         params = {k: str(v) for k, v in params.items() if v is not None}
 
-        headers = {
-            "accept": "application/json",
-            "x-auth-token": self.client.secret_key or "",
-            "user-agent": self.client.user_agent,
```

Comment:
This might introduce a breaking change since any updates made to the client attributes after initialization won't be reflected into the session